### PR TITLE
Initial Commit v0.1.0

### PR DESCRIPTION
**Pull request by dmaloneycalu**
Copied and tidied up code from existing project. Created a roadmap for where it should go.

After this is successfully merged/tagged, we can release it through Bower (allowing us to reference it as just a version number and not its github address). Any and all thoughts appreciated.

@StoicJester @jrit @Tathanen
@scottferg

### CODE REVIEW

**Comment by scottferg** [comment]
Needs an open source license and the proper heading on source files. MIT or BSD will suffice.

File: svg-radial.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            transclude: true,
+            replace: true,
+            template:   '<div class="radial-progress">' +
+                            '<div class="base"></div>' +
+                            '<div class="radial">' +
+                                '<div class="piece"></div>' +
+                                '<div class="piece"></div>' +
+                            '</div>' +
+                            '<div class="transclude" ng-transclude></div>' +
+                        '</div',
```
Comment (Tathanen) [inline svg-radial.js]: Missing >?

Think about using a template file instead, too.

File: less/svg-radial.less [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    clip: rect( @t, @r, @b, @l );
+}
+.transform-transition( @time )
+{
+    -webkit-transition: -webkit-transform @time ease-out;
+    -moz-transition:    -moz-transform @time ease-out;
+    -o-transition:      -o-transform @time ease-out;
+    -ms-transition:     -ms-transform @time ease-out;
+    transition:         transform @time ease-out;
+}
```
Comment (Tathanen) [inline less/svg-radial.less]: Wow that was a lot of variables.

File: svg-radial.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+"use strict";
```
Comment (jrit) [inline svg-radial.js]: maybe move this out of global scope for concat friendliness

File: svg-radial.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                    angle1 = angle2 = 360 * percent / 100;
+                    //make sure that the first piece doesn't rotate too far
+                    angle1 = Math.min( 180, angle1 );
+                    //this next line is to prevent bugs if the entered percentage is over 100
+                    angle2 = Math.min( 360, angle2 );
+
+                    setRotation( ".piece:first-child", angle1 );
+                    setRotation( ".piece:last-child", angle2 );
+
+                    element.find( ".radial" ).toggleClass( "gtFifty", ( percent > 50 ) );
```
Comment (Tathanen) [inline svg-radial.js]: I believe finding with selectors only works if you include full jQuery.  So if you wanna do that, make jQuery a dependency in bower.json.

File: svg-radial.js (hunk 3 above)
Comment (Tathanen) [inline svg-radial.js]: Seconded.

File: less/svg-radial.less [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        @radial-progress-transclude-buffer: 0,
+        @radial-progress-border-color: #8BC53E,
+        @radial-progress-border-base-color: #C3C3C3 ) {
+    
+    @radial-progress-radius: @radial-progress-diameter / 2;
+    @radial-progress-transclude-size: @radial-progress-diameter -
+        @radial-progress-border-size -
+        @radial-progress-border-size -
+        @radial-progress-transclude-buffer -
+        @radial-progress-transclude-buffer;
```
Comment (jrit) [inline less/svg-radial.less]: some mixed indents

File: svg-radial.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+angular.module( "svgRadial", [] )
+    .directive( "radialProgress", function ()
+    {
+        return {
+            restrict: "E",
```
Comment (Tathanen) [inline svg-radial.js]: Any reason in particular that you want it restricted?

File: less/svg-radial.less [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    .transclude {
+        position: absolute;
+        top: @radial-progress-border-size + @radial-progress-transclude-buffer;
+        left: @radial-progress-border-size + @radial-progress-transclude-buffer;
+        height: @radial-progress-transclude-size;
+        width: @radial-progress-transclude-size;
+        border-radius: 50%;
+        overflow: hidden;
+
+
```
Comment (Tathanen) [inline less/svg-radial.less]: Superfluous second blank line.

File: svg-radial.js (hunk 6 above)
Comment (dmaloneycalu) [inline svg-radial.js]: There's no particular reason for this yet. It was just how it was included in our project at the time. All of this is mostly copy/paste.

File: less/svg-radial.less (hunk 5 above)
Comment (StoicJester) [inline less/svg-radial.less]: Those lines are indented because they are actually being subtracted from the line above

File: svg-radial.js (hunk 4 above)
Comment (StoicJester) [inline svg-radial.js]: yup, limited to tag name finding

File: less/svg-radial.less (hunk 5 above)
Comment (jrit) [inline less/svg-radial.less]: so they are, we usually have the operators at the start of the lines and was "operator blind"

File: svg-radial.js (hunk 4 above)
Comment (dmaloneycalu) [inline svg-radial.js]: Good catch, will include that in the bower file. Adding it to the roadmap to remove that dependency by v1.

**Comment by dmaloneycalu** [comment]
@Tathanen @jrit 

- Added jQuery as dependency
- Closed the div tag
- Move the `use strict` statement

@scottferg 

- Added `license.md` and a license block to the two source files

**Comment by scottferg** [comment]
All caps, `LICENSE.md` then LGTM

File: license.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Vokal
```
Comment (scottferg) [inline license.md]: Vokal LLC

**Comment by dmaloneycalu** [comment]
@jrit @Tathanen @StoicJester Any last notes?

@scottferg completed the LLC changes

**Comment by dmaloneycalu** [comment]
Bump :+1: :-1:

**Comment by Tathanen** [comment]
LGTM

**Pull request merged**